integration-libs/opf/checkout/components/opf-checkout-review-cart-details/opf-checkout-review-cart-details.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="cx-review-cart-total d-none d-lg-block d-xl-block">
+  {{
+    'cartItems.cartTotal' | cxTranslate: { count: cart?.deliveryItemsQuantity }
+  }}:
+  {{ cart?.totalPrice?.formattedValue }}
+</div>
+<div class="cx-review-cart-item col-md-12">
+  <cx-promotions
+    [promotions]="
+      (cart?.appliedOrderPromotions || []).concat(
```

Comment:
Since the component is OnPush, this calculation will only be done on cart change, right?

Reply:
That's correct. It's in an OnPush component, so the calculation only happens on cart changes.
The template binding is optimized by Angular's change detection.